src/postman/util.ts

```diff
@@ -1,4 +1,50 @@
-import { Item, ItemGroup } from 'postman-collection';
+import { IHttpParam } from '@stoplight/types';
+import { JSONSchema4 } from 'json-schema';
+import { DescriptionDefinition, Item, ItemGroup } from 'postman-collection';
+import * as toJsonSchema from 'to-json-schema';
+import { JSONSchema3or4 } from 'to-json-schema';
+
+export function transformValueTemplateToSchema(value: string): JSONSchema4 | undefined {
+  switch (value) {
+    case '<string>':
+      return { type: 'string' };
+    case '<long>':
+      return { type: 'integer' };
+  }
+
+  return;
+}
+
+export function transformValueToHttpParam(value: string): Pick<IHttpParam, 'schema' | 'examples'> {
```

Comment:
This function is not really returning an Http Param, it's rather extracting an example or returning a schema — can we reword the name?

Reply:
Hm, you right it is not HttpParam. Hard to say what it is but `..Schema` is a better name. Updated.